libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts

```diff
@@ -0,0 +1,153 @@
+import { importProvidersFrom } from '@angular/core';
+
+const props = {
+  table: {
+    category: 'template-options',
+  },
+};
+
+export default {
+  title: 'Formly/Stepper',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        ReactiveFormsModule,
+        SdsFormlyModule,
+        FormsModule,
+        FormlyStepperIntroductionModule,
+        FormlyStepperBasicModule,
+        FormlyStepperAdvancedModule,
+        UswdsStepperModule,
+      ],
+    }),
+    applicationConfig({
+      providers: [provideAnimations(), importProvidersFrom(FormlyModule.forRoot())],
+    }),
+  ],
+} as Meta;
+
+export const Introduction: StoryObj = (args) => ({
+  template: '<sds-formly-stepper-introduction></sds-formly-stepper-introduction>',
+  props: args,
+});
+Introduction.parameters = {
+  controls: { disable: true },
+  actions: { disable: true },
+  sdsCodePreview: { disable: true },
+};
+
+export const Uneven: StoryObj = (args) => ({
+  template: '<sds-formly-stepper-basic></sds-formly-stepper-basic>',
+  props: args,
+});
+Uneven.parameters = {
+  controls: {
+    disable: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/formly/formly-stepper/formly-stepper-basic',
+    'FormlyStepperBasicModule',
+    'sds-formly-stepper-basic',
+    [
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.ts',
+        'ts',
+        false
+      ),
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'basic'),
+};
+
+export const USWDS: StoryObj = (args) => ({
+  template: '<gsa-sam-uswds-stepper></gsa-sam-uswds-stepper>',
+  props: args,
+});
+USWDS.parameters = {
+  controls: {
+    disable: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/formly/formly-stepper/formly-stepper-uswds',
+    'UswdsStepperModule',
+    'gsa-sam-uswds-stepper',
+    [
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-uswds/formly-stepper-uswds-custom.component.ts',
+        'ts',
+        false
+      ),
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-uswds/formly-stepper-uswds-custom.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'uswds'),
+};
+
+export const Advanced: StoryObj = (args) => ({
+  template: '<stepper-advanced-demo></stepper-advanced-demo>',
+  props: args,
+});
+Advanced.parameters = {
+  controls: {
+    disable: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
```

Comment:
Need to update the key for actions as well

Reply:
done